### PR TITLE
Feature/deploy with clm

### PR DESCRIPTION
**Pull request by dmpop**
This pull request combines the Planning and Installation Guides into a single Deployment Guide with CLM. It also removes obsolete DC files and introduces the `DC-suse-openstack-cloud-deploy-clm` file for generating the unified Guide.

### CODE REVIEW

File: xml/entity-decl.ent [hunk 1]
```diff
@@ -189,6 +189,7 @@ suggested official replacement from rsalevsky. -->
 <!ENTITY soclcminstall  "<citetitle xmlns='http://docbook.org/ns/docbook'>Installing with &lcm;</citetitle>">
 <!ENTITY rhel-install  "<citetitle xmlns='http://docbook.org/ns/docbook'>Installing &rhla;</citetitle>">
 <!ENTITY soclcmplan     "<citetitle xmlns='http://docbook.org/ns/docbook'>Planning an Installation with &lcm;</citetitle>">
+<!ENTITY soclcmdeploy     "<citetitle xmlns='http://docbook.org/ns/docbook'>Deployment Guide with &lcm;</citetitle>">
```
Comment (asettle) [inline xml/entity-decl.ent]: Only one nit: The other guide is Deployment Guide *using* < product > 

We should make it consistent.

File: xml/planning-planning-planning_index.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 ]>
 <part xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
```
Comment (asettle) [inline xml/planning-planning-planning_index.xml]: Can this file be renamed?

File: xml/installation-installation-installation_overview.xml [hunk 3]
```diff
@@ -15,7 +15,7 @@
   <listitem>
    <para>
     Work through the information in the <xref
```
Comment (asettle) [inline xml/installation-installation-installation_overview.xml]: Ditto - these file names are *whack*

**Comment by dmpop** [comment]
I did my best to avoid drastic changes to the underlying structure.

**Pull request closed without merging**